spec/shelter_spec.rb

```diff
@@ -50,4 +50,21 @@
         expect(shelter.call_pets).to eq(['Salem!', 'Beethoven!', 'Spot!', 'Jonesy!'])
       end
     end
-end
\ No newline at end of file
+
+  describe '#over_capacity' do
+      it 'returns true if shelter is over listed capacity' do
```

Comment:
This test looks great!  Awesome job making sure that you have all of the possible branches (t/f) tested!